src/SdfGenerator.hh

```diff
@@ -44,6 +44,7 @@ namespace sdf_generator
   /// \input[in] _config Configuration for the world generator
   /// \returns Generated world string if generation succeeded.
   /// Otherwise, nullopt
+  IGNITION_GAZEBO_VISIBLE
```

Comment:
Same about the header not being installed.

Reply:
The header is not installed but it host declarations of functions that will go into ignition-gazebo5.lib. Once the binary symbols are generated and go in that lib any external binary/lib using the functions will need them to be exported, in this case SdfGenerator_TEST. So yes, we need these ones.